src/main/java/seedu/application/logic/parser/EditCommandParser.java

```diff
@@ -48,6 +46,10 @@ public EditCommand parse(String args) throws ParseException {
             editJobDescriptor.setDeadline(ParserUtil.parseDeadline(argMultimap.getValue(PREFIX_DEADLINE).get()));
         }
 
+        if (argMultimap.getValue(PREFIX_STATUS).isPresent()) {
```

Comment:
Reminder to add test case to cover this line. Probably under EditCommandParserTest